www/js/utils.js

```diff
@@ -2,7 +2,7 @@ var utils = angular.module('starter.controllers.utils', []);
 
 // creates a list of constants that are accessible anywhere
 utils.constant('constants', {
-    'BASE_SERVER_URL': 'http://ec2-52-91-208-65.compute-1.amazonaws.com:3002/api/',
+    'BASE_SERVER_URL': 'http://ec2-52-91-208-65.compute-1.amazonaws.com:3003/api/',
```

Comment:
I'm not exactly sure what you mean by this. This is the only endpoint that we have running that currently has the fix so that pre-flight works properly for signin. We either need to add this code to the other endpoints or change to this endpoint (what I did here).